### DISCUSSION TITLE
[Enhancement] [tree-select] Cascader 和 TreeSelect 在多选框情况下希望最后一级无子节点的节点可通过直接点击 label 进行选中

### DISCUSSION THREAD

**Discussion by crateink** [element-plus/element-plus#19203] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tree-select`

## Description

目前最后一级必须点击前面多选框才能选中，antdesign目前已经支持最后一级点击label选中

### Additional comments

_(empty)_

**Comment by Dsaquel** [comment]
@Netuoin If you want to select at any level of the tree/cascader then checkout [TreeSelect docs](https://element-plus.org/zh-CN/component/tree-select.html#%E9%80%89%E6%8B%A9%E4%BB%BB%E6%84%8F%E7%BA%A7%E5%88%AB)
& [Cascader docs](https://element-plus.org/zh-CN/component/cascader.html#%E9%80%89%E6%8B%A9%E4%BB%BB%E6%84%8F%E4%B8%80%E7%BA%A7%E9%80%89%E9%A1%B9) 
By default Ant Design handle this behavior. In Element plus you need to specify `checkStrictly: true` to activate any level selection.

**Reply by crateink** [reply]
I think you misunderstood what I meant. I meant that the last level of selection requires clicking the checkbox in front of the label, which is not very convenient for operation.

**Reply by Dsaquel** [reply]
@netuoin 
Agree in `show-checkbox` mode we should be able to select a node if they dont have any children.
But this will be maybe an unexpected behavior because when `show-checkbox` is true then `check-on-click-node` is false too.
Im quite busy this time but i can work on a PR in the next weeks.
